modules/claims_api/lib/claims_api/v2/error/form_526_error_mapper.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class Form526ErrorMapper
+    FORM_526_ERROR_DICTIONARY = {
+      InProcess: %w['This claim is already in progress'],
+      marshalError: %w['A marshalling error occurred.'],
+      startBatchJobError: %w['There was an internal error'],
+      submit_establishClaim_serviceError: %w['The claim failed to establish'],
+      submit_load_vaOffice_serviceError: %w['VA Office service error'],
+      submit_load_benefitClaim_serviceError: %w[A service error occurre while loading the claim],
+      submit_save_draftForm_serviceError: %w[A service error occurred during the claim submission],
+      submit_save_draftForm_PIFInUse: %w[There was a problem with the draft form which was already in use],
+      submit: %w['The claim could not be established'],
+      disabled: %w['this claim has been disabled'],
+      submit_save_draftForm_MaxEPCode: %w['This claim could not be established. The Maximum number of EP codes have been reached for this benefit type claim code'], # rubocop:disable Layout/LineLength
+      submit_noRetryError: %w['This job is no longer able to be re-tried'],
+      header_va_eauth_birlsfilenumber_Invalid: %w[There is a problem with your birls file number please contact...]
+    }.freeze
+
+    def initialize(error)
+      @error = error
+    end
+
+    def get_details
+      key = if @error[:key].include?('form526')
+              @error[:key].slice!('form526').gsub('.', '_')
+            else
+              @error[:key].gsub('.', '_')
+            end
+      err_info = @error[:detail] || @error[:text]
+      FORM_526_ERROR_DICTIONARY[key.to_sym].presence ||
+        "The claim could not be established - #{err_info}. Hint: #{@error[:key]}"
+    end
+  end
+end
```

Comment:
Wondering if this would only ever be used for form 526?  When I initially did that `lighthouse_error_handler` stuff I had named it something like this, but then realized it would be covering other areas possibly.  I'm not sure about the discussions on this so if this is truly only for 526 then I can resolve this but if there is a possibility for this to used in POA then I think we may want to reconsider the name here.

Reply:
@rockwellwindsor-va Thanks, that is a good consideration! what do you think of `DockerContainerErrorMapper`? or  do you have another idea?

Reply:
@stiehlrod  I was kind of thinking the _less_ specific the better.  Looking at this more closely though it does look like this is only loaded into the custom_error which is handling the claims stuff.  If you think there is a possibility this could be extended at some point into other jobs like POA or other areas then I would recommend `lighthouse_error_mapper` as a way to make it more easily extendable but in its current state I think the name works, or it could even be something `claims_error_mapper`.  Just worried that if it goes anywhere other then 526, there is a lot of names that have to be  changed.